tests/python/tank_test/tank_test_base.py

```diff
@@ -686,7 +779,7 @@ def add_to_sg_mock_db(self, entities):
                     # print "Swapping link dict %s -> %s" % (entity[x], link_dict)
                     entity[x] = link_dict
 
-            self.tk.shotgun._db[et][eid] = entity
+            self.mockgun._db[et][eid] = entity
```

Comment:
Seems to answer one of my earlier questions: `self.tk.shotgun` was always a `mockgun` wasn't it? So it was already not possible to run these tests against a real SG server?

Reply:
Yup. Our tests often expects certain ids being returned, because we know everything is starting from 1 every single time. It would be quite hard to port the tests I think.